docs/conversion_example_gallery.rst

```diff
@@ -29,3 +29,26 @@ Sorting
     Kilosort <conversion_examples_gallery/sorting/kilosort>
     Neuroscope <conversion_examples_gallery/sorting/neuroscope>
     Phy <conversion_examples_gallery/sorting/phy>
+
+
+Imaging
+-------
+
+.. toctree::
+    :maxdepth: 1
+
+    Tiff <conversion_examples_gallery/imaging/tiff>
+    HDF5 <conversion_examples_gallery/imaging/hdf5imaging>
+    Scanbox <conversion_examples_gallery/imaging/scanbox>
+    ScanImage -TBD
+
+Segmentation
+------------
+
+.. toctree::
+    :maxdepth: 1
+
+    Suit2P (TBD)
+    Caiman (TBD)
+    Extraction (TBD)
+    Cnmfe (Tbd)
```

Comment:
```suggestion
    CNMFE (Tbd)
```

Abbreviation for Constrained Nonnegative Matrix Factorization for microEndoscopic data (https://github.com/zhoupc/CNMF_E), both this and EXTRACT are two forms of analysis from the Schnitzer lab (https://github.com/schnitzer-lab/nwbpkg/blob/88a89e8bb0a6aaca1234a3647676ae221666f60b/%2Bnwbpkg/mat2nwb.m#L16-L17)